functions/__format_time.fish

```diff
@@ -1,9 +1,9 @@
 function __format_time -d "Format milliseconds to a human readable format"
   set -l milliseconds $argv[1]
-  set -l seconds (math "$milliseconds / 1000 % 60")
-  set -l minutes (math "$milliseconds / 60000 % 60")
-  set -l hours (math "$milliseconds / 3600000 % 24")
-  set -l days (math "$milliseconds / 86400000")
+  set -l seconds (math -s0 "$milliseconds / 1000 % 60")
```

Comment:
Do you mind using long option format in order to make code less cryptic and easier to contribute to?


```suggestion
  set -l seconds (math --scale=0 "$milliseconds / 1000 % 60")
```

Reply:
We shouldn't, because fish 2.4.0's math actually does not support that. So that would bump the required version to 2.7.0.